system-x/services/jms/rabbitmq/src/main/java/software/tnb/jms/rabbitmq/service/RabbitMQ.java

```diff
@@ -0,0 +1,26 @@
+package software.tnb.jms.rabbitmq.service;
+
+import software.tnb.common.service.Service;
+import software.tnb.jms.rabbitmq.account.RabbitmqAccount;
+import software.tnb.jms.rabbitmq.validation.RabbitMQValidation;
+
+import javax.jms.Connection;
+
+public abstract class RabbitMQ extends Service<RabbitmqAccount, Connection, RabbitMQValidation> {
+    public static final int PORT = 5672;
+    public static final int MANAGEMENT_PORT = 15672;
+    public static final String IMAGE = "quay.io/rh_integration/rabbitmq:3-management";
+
+    public abstract String brokerUrl();
+
+    protected abstract String getServerUrl();
```

Comment:
do we need both? I think `getServerUrl()` is enough

Reply:
@mcarlett AMQ service uses similar approach and I use brokerUrl() in the tests. In case I would need to add it to Validation class.